docs/serializations.md

```diff
@@ -33,4 +33,28 @@ Canonical serialization is in JSON format, as defined in RFC 8259 (IETF STD 90),
 - integers: represented in base 10 using decimal digits. This designates an integer component that may be prefixed with an optional minus sign. Leading zeros are not allowed.
 - strings: UTF-8 representation without specific canonicalisation. A string begins and ends with quotation marks (%x22). Any Unicode characters may be placed within the quotation marks, except for the two characters that MUST be escaped by a reverse solidus: quotation mark, reverse solidus, and the control characters (U+0000 through U+001F).
 - arrays: An array structure is represented as square brackets surrounding zero or more items. Items are separated by commas.
-- objects: An object structure is represented as a pair of curly brackets surrounding zero or more name/value pairs (or members). A name is a string containing only ASCII characters (0x21-0x7F). The names within an object must be unique. A single colon comes after each name, separating the name from the value. A single comma separates a value from a following name. The name/value pairs are ordered by name.
\ No newline at end of file
+- objects: An object structure is represented as a pair of curly brackets surrounding zero or more name/value pairs (or members). A name is a string containing only ASCII characters (0x21-0x7F). The names within an object must be unique. A single colon comes after each name, separating the name from the value. A single comma separates a value from a following name. The name/value pairs are ordered by name.
+
+## 4.4 Serialization information
+
+A collection of elements may be serialized in multiple formats.
+
+An SpdxDocument element represents the common properties of a collection of
```

Comment:
What does "common properties" mean in this context? An SpdxDocument is a collection of things, why do we refer to "common" properties?

Reply:
Trying to go through the history and meeting minutes, it's from PR #509 based on [2023-11-07 Tech meeting](https://github.com/spdx/meetings/blob/main/tech/2023/2023-11-07.md). Cannot find an explanation.

I would guess from the sentence that there can be a case that some serialization data formats cannot fully store all the properties? Say there are 3 serialization data formats: A, B, C. A and B able to serialized everything, but C can only do some (there's a data loss). So in this case, an SpdxDocument element can represents only "common properties" that A and B and C can serialized?

I've limited expertise in this area anyway. May need to ping someone from that meeting.